skills/honcho-setup/SKILL.md

```
@@ -0,0 +1,98 @@
+**Directories to check:**
+- `memory/` — recursively read all files inside
+- `canvas/` — recursively read all files inside
+
+Files found inside `memory/` and `canvas/` are treated as user/owner content.
+
+### Upload to Honcho
+
+Check that `HONCHO_API_KEY` is set (it may be in `~/.openclaw/.env`). If it's not set, warn the user:
+
+> HONCHO_API_KEY is not set. Set it with:
+> ```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)